litrpc/lit-accounts.proto

```diff
@@ -144,7 +144,10 @@ message UpdateAccountRequest {
     // The ID of the account to update. Either the ID or the label must be set.
     string id = 1;
 
-    // The new account balance to set. Set to -1 to not update the balance.
+    /*
+    Deprecated, use credit or debit command instead. The new account balance to
+    set. Set to -1 to not update the balance.
```

Comment:
There is some deprecation functionality for proto files, which we could use in addition:
`int64 account_balance = 2 [deprecated = true];`